library/src/commonTest/kotlin/com/mercury/sqkon/TestDataClasses.kt

```diff
@@ -93,6 +93,7 @@ data class TypeTwoData(
     val otherValue: Int
 )
 
+@Serializable
```

Comment:
@chrisjenx I think this is what you were running into with `@SerialName`; enums require `@Serializable` or else `@SerialName` is ignored.

Reply:
Yes, to which the Serialization team still need to fix/follow up